src/Umbraco.Core/Sync/RefreshMethodType.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+
+namespace Umbraco.Core.Sync
+{
+    /// <summary>
+    /// Describes <see cref="RefreshInstruction"/> refresh action type.
+    /// </summary>
+    [Serializable]
+    public enum RefreshMethodType
+    {
+        // NOTE
+        // that enum should get merged somehow with MessageType and renamed somehow
+        // but at the moment it is exposed in CacheRefresher webservice through RefreshInstruction
+        // so for the time being we keep it as-is for backward compatibility reasons
+
+        RefreshAll,
+        RefreshByGuid,
+        RefreshById,
+        RefreshByIds,
+        RefreshByJson,
+        RemoveById,
+
+        // would adding values break backward compatibility?
```

Comment:
@zpqrtbnk  No, adding message types would *not* be breaking, so we should go ahead and add whatever is necessary